### PR TITLE
Fix: Remove the ignore errors of `HTTP` in phpstan.

### PR DESCRIPTION
**Pull request by ya861220**
**Description**
See #7731 

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [ ] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

File: system/HTTP/DownloadResponse.php [hunk 1]
```diff
@@ -128,6 +132,8 @@ public function getContentLength(): int
 
     /**
      * Set content type by guessing mime type from file extension
+     *
+     * @return void
      */
     private function setContentTypeByMimeType()
```
Comment (datamweb) [inline system/HTTP/DownloadResponse.php:138]:
```suggestion
     */
    private function setContentTypeByMimeType(): void
```
see https://github.com/codeigniter4/CodeIgniter4/pull/7794#discussion_r1286958733

**Review by datamweb — CHANGES_REQUESTED** [review]

File: system/HTTP/DownloadResponse.php (hunk 1 above)
Comment (ya861220) [reply system/HTTP/DownloadResponse.php]: Ok! Done.

**Comment by kenjis** [comment]
Please rebase to resolve conflicts.
See https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/workflow.md#updating-your-branch

File: system/HTTP/Files/UploadedFileInterface.php [hunk 2]
```diff
@@ -59,6 +59,8 @@ public function __construct(string $path, string $originalName, ?string $mimeTyp
      * @param string $targetPath Path to which to move the uploaded file.
      * @param string $name       the name to rename the file to.
      *
+     * @return void
```
Comment (kenjis) [inline system/HTTP/Files/UploadedFileInterface.php:62]:
```suggestion
     * @return bool
```

Comment (ya861220) [reply system/HTTP/Files/UploadedFileInterface.php]: Ok! Done.

**Review by kenjis — APPROVED** [review]

**Review by MGatner — APPROVED** [review]

**Comment by MGatner** [comment]
Thank you!

**Pull request merged**